ci/jenkins/Jenkins_steps.groovy

```diff
@@ -825,7 +825,21 @@ def test_unix_scala_cpu() {
       node(NODE_LINUX_CPU) {
         ws('workspace/ut-scala-cpu') {
           timeout(time: max_time, unit: 'MINUTES') {
-            utils.unpack_and_init('cpu', mx_dist_lib, true)
+            utils.unpack_and_init('cpu', mx_lib, true)
```

Comment:
I don't think we need mx_dist_lib anymore. If so, we should just delete the variable